lib/feed.xml

```diff
@@ -53,26 +53,12 @@
       <content type="html" xml:base="{{ post.url | prepend: url_base | xml_escape }}">{{ post.content | strip | xml_escape }}</content>
 
       {% assign post_author = post.author | default: post.authors[0] | default: site.author %}
-      {% assign post_author_email = false %}
-      {% assign post_author_uri = false %}
+      {% assign post_author = site.data.authors[post_author] | default: post_author %}
```

Comment:
`post.author` can be a string or a hash. It's kind of a mess ¯\_(ツ)_/¯

If we changed _this_ line, we would also need to change the line above it to `post_author_data`, but then it would seem weird to `{% assign post_author_data = site.data.authors[post_author_data] | default: post_author_data %}`

I am open to suggestion though.